service/lxd.go

```diff
@@ -309,6 +309,14 @@ func (s *LXDService) HasExtension(ctx context.Context, target string, address st
 		}
 	}
 
+	// Fill the cache of API extensions.
+	// If the client's internal `server` field isn't yet populated
+	// a call to HasExtension will always return true for any extension.
```

Comment:
Is there a corresponding fix for this in LXD?

Reply:
As we discussed earlier I'll follow up on this in LXD.